docs/reference/configuration.md

```diff
@@ -0,0 +1,1067 @@
+---
+mapped_pages:
+  - https://www.elastic.co/guide/en/apm/agent/python/current/configuration.html
+---
+
+# Configuration [configuration]
+
+To adapt the Elastic APM agent to your needs, configure it using environment variables or framework-specific configuration.
+
+You can either configure the agent by setting environment variables:
+
+```bash
+ELASTIC_APM_SERVICE_NAME=foo python manage.py runserver
+```
+
+or with inline configuration:
+
+```python
+apm_client = Client(service_name="foo")
+```
+
+or by using framework specific configuration e.g. in your Django `settings.py` file:
+
+```python
+ELASTIC_APM = {
+    "SERVICE_NAME": "foo",
+}
+```
+
+The precedence is as follows:
+
+* [Central configuration](#config-central_config) (supported options are marked with [![dynamic config](../images/dynamic-config.svg "") ](#dynamic-configuration))
```

Comment:
Rendering of this badge looks off, not a blocker ofc

![Schermata del 2025-03-04 16-59-39](https://github.com/user-attachments/assets/3e9c9d30-08b1-4e94-8126-07bfeb710f0f)

Reply:
This is a known issue: https://github.com/elastic/docs-builder/issues/601